IBPSA/Fluid/FixedResistances/Validation/PlugFlowPipes/PlugFlowAIT.mo

```diff
@@ -261,6 +255,36 @@ model PlugFlowAIT
         rotation=180,
         origin={46,56})));
 
+  IBPSA.Fluid.FixedResistances.Junction splPip1(
+    redeclare package Medium = Medium,
+    m_flow_nominal={pip1.m_flow_nominal,pip5.m_flow_nominal,pip4.m_flow_nominal},
+    from_dp=true,
+    linearized=true,
```

Comment:
this has no purpose if `dp_nominal=0`?

Reply:
Indeed! Good catch. Removed for both junctions in this model.